### PR TITLE
Implements MsSummaryCard

### PR DESCRIPTION
**Pull request by NicoTuxx**

### CODE REVIEW

File: lib/components/ms-card/MsSummaryCardItem.vue [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      v-show="error"
+      class="form-error body"
+    >
+      {{ $msTranslate(error) }}
+    </p>
+    <div
+      v-if="secondLine"
+      class="ms-summary-card-item__text subtitles-normal"
+    >
+      {{ secondLine.text }}
```
Comment (Max-7) [inline lib/components/ms-card/MsSummaryCardItem.vue:15]: Should be translated.

File: lib/components/ms-card/types.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { Translatable } from '@lib/services';
+
+interface MsSummaryCardItemData {
+  label: Translatable;
+  text: string;
```
Comment (Max-7) [inline lib/components/ms-card/types.ts:5]: Should be a Translatable, not a string.

File: lib/components/ms-card/types.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  error?: Translatable;
+  secondLine?: {
+    text: string;
```
Comment (Max-7) [inline lib/components/ms-card/types.ts:8]: Translatable.

**Review by Max-7 — CHANGES_REQUESTED** [review]

File: lib/components/ms-card/MsSummaryCardItem.vue (hunk 1 above)
Comment (NicoTuxx) [reply lib/components/ms-card/MsSummaryCardItem.vue:15]: Nope, we pass value here not a label, could be anything coming from user input, server data...

File: lib/components/ms-card/types.ts (hunk 2 above)
Comment (NicoTuxx) [reply lib/components/ms-card/types.ts:5]: Same.

File: lib/components/ms-card/types.ts (hunk 3 above)
Comment (NicoTuxx) [reply lib/components/ms-card/types.ts:8]: Same.

**Review by Max-7 — APPROVED** [review]

**Pull request merged**